build/commands/lib/config.js

```diff
@@ -409,9 +409,14 @@ Config.prototype.buildArgs = function () {
       args.use_system_xcode = false
   }
 
-  if (this.getTargetOS() === 'linux' && this.targetArch === 'x64') {
-    // Include vaapi support
-    args.use_vaapi = true
+  if (this.getTargetOS() === 'linux') {
+    if (this.targetArch === 'x64') {
+      // Include vaapi support
+      args.use_vaapi = true
```

Comment:
do we not need vaapi for arm as well? https://github.com/brave/brave-core/commit/87d58be41ea902cd55455e47c7c5ef2a68b47c3c

Reply:
I enabled it now for Arm64 in 32ca54f91931f397ff1dce3319a2afc6a68c148b. I don't have Arm64 hardware with a GPU to test it. But the build passed and Brave launches without issues for me on Arm64 Ubuntu (without a GPU). Since vaapi needs to be enabled with a runtime flag anyways, I believe it's OK to set `use_vaapi` for Arm64 as well.

I also added a comment to investigate enabling it for x86 and via `use_vaapi_x11` instead of `use_vaapi` in 5651fab1f92065d97173d491f323b0b5f8f24e56.